bartop-ui/src/config.js

```diff
@@ -9,9 +9,13 @@ const requiredEnvVars = [
 const missingEnvVars = requiredEnvVars.filter(envVar => !process.env[envVar]);
 
 if (missingEnvVars.length > 0) {
-  throw new Error(
-    `Missing necessary environment variables: ${missingEnvVars.join(', ')}`
-  );
+  const message = `Missing necessary environment variables: ${missingEnvVars.join(
+    ', '
+  )}`;
+  if (process.env.NODE_ENV === 'PRODUCTION') {
```

Comment:
Developing in Storybook doesn't require the env vars, but still imports this file and breaks w/o them. So I switched it to just warning about missing env vars if it's not production